Resources/Prototypes/_Goobstation/Recipes/Lathes/devices.yml

```
@@ -27,3 +29,27 @@
+
+- type: latheRecipe
+  id: ProximitySignaller
+  result: ProximitySignaller
+  categories:
+  - Parts
+  completetime: 2
+  materials:
+    Steel: 100
+    Plastic: 200
+    Glass: 100
+    Gold: 200
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)